Missing os dependency in cypress/browsers images

The cypress docs recommend using `start-server-and-test` (https://docs.cypress.io/guides/continuous-integration/introduction#Boot-your-server)
When following this advice with your image: `cypress/browsers:node14.17.6-chrome100-ff98`
The test run fine, but at the very end it always fails with:
```
events.js:377
      throw er; // Unhandled 'error' event
      ^
Error: spawn ps ENOENT
    at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21)
Emitted 'error' event on ChildProcess instance at:
    at Process.ChildProcess._handle.onexit (internal/child_process.js:275:12)
    at onErrorNT (internal/child_process.js:467:16)
    at processTicksAndRejections (internal/process/task_queues.js:82:21) {
  errno: -2,
  code: 'ENOENT',
  syscall: 'spawn ps',
  path: 'ps',
  spawnargs: [ '-A', '-o', 'ppid,pid,stat,comm' ]
}
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! wen-ui@0.26.1 e2e:ci: `start-server-and-test e2e:ci:serve http://localhost:4200/ e2e:ci:run`
npm ERR! Exit status 1
```

And according to my research, the command line utility `ps` is used there but it is not pre-installed in your image (see https://github.com/bahmutov/start-server-and-test/issues/132#issuecomment-448581335)

--- a/scripts/generate-base-image.js
+++ b/scripts/generate-base-image.js
@@ -51,6 +51,7 @@ RUN apt-get update && \\
   libxss1 \\
   libasound2 \\
   libxtst6 \\
+  ps \\
   xauth \\
   xvfb \\
   # install text editors

--- a/scripts/generate-base-image.js
+++ b/scripts/generate-base-image.js
@@ -51,7 +51,7 @@ RUN apt-get update && \\
   libxss1 \\
   libasound2 \\
   libxtst6 \\
-  ps \\
+  procps \\
   xauth \\
   xvfb \\
   # install text editors

